packages/palette/src/elements/Toggle/__tests__/Toggle.test.tsx

```diff
@@ -38,13 +39,22 @@ describe("Toggle", () => {
   })
 
   it("defaults chevron size to 12px", () => {
-    const wrapper = mount(
-      <Toggle>
-        tab content
-      </Toggle>
-    )
+    const wrapper = mount(<Toggle>tab content</Toggle>)
     const chevron = wrapper.find(ChevronIcon)
     expect(chevron.props().width).toEqual(12)
     expect(chevron.props().height).toEqual(12)
   })
+
+  it("renders proper component when passed component as label", () => {
+    const component = <Text>Hello</Text>
+    const wrapper = mount(<Toggle label={component}>tab content</Toggle>)
+    expect(wrapper.find("Text")).toHaveLength(1)
+  })
+
+  it.only("renders Sans component when passed string as label", () => {
```

Comment:
You'll want to remove this `.only` (We need to install a linting rule that catches these pre-commit)

Reply:
Oh yes thanks for catching!